### DISCUSSION TITLE
can't print in calc since 8.2.1

### DISCUSSION THREAD

**Discussion by bwragg** [nextcloud/all-in-one#4705] [Questions] [answered]
Hi,

Our site has had an issue where calc won't print since 8.2.1 . We found this is a bug in Collabora that is fixed and will be added to 24.04.3 https://github.com/CollaboraOnline/online/pull/9070

So from what I've read I can't selectively update to this because AIO builds the source code, so is there a way to apply this fix? 

Thanks,

Benjamin

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/pull/4691
